lib/oaken/railtie.rb

```diff
@@ -1,5 +1,11 @@
-class Oaken::Railtie < Rails::Railtie
-  initializer "oaken.lookup_paths" do
-    Oaken.lookup_paths << "db/seeds/#{Rails.env}"
+module Oaken
+  class Railtie < Rails::Railtie
+    config.app_generators do
+      _1.test_framework _1.test_framework, fixture: false
+    end
```

Comment:
I dug around in the Rails internals and we can simplify the previous commit down to this. I'm still not sure if we should ship this in Oaken because of upgrading apps.